spec/mailers/notification_mailer_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe NotificationMailer, type: :mailer do
+  describe 'Notification email' do
+    TEMPLATE_URL = '../../spec/fixtures/notification_mailer/test_email.html.slim'
+    let(:user) { create(:user, name: 'tester') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.